app/scripts/services/validateRegistrant.js

```diff
@@ -3,44 +3,54 @@
 angular.module('confRegistrationWebApp')
   .service('validateRegistrant', function validateRegistrant() {
 
-    var blockVisibleRuleCheck = function(block, registrant){
-      var returnValue = true;
+    var blockVisibleRuleCheck = function (block, registrant) {
       var answers = registrant.answers;
-      angular.forEach(block.rules, function(rule){
-        var answer = _.find(answers, {blockId: rule.parentBlockId});
-        if(angular.isUndefined(answer) || answer.value === ''){
-          returnValue = false;
-        }else{
-          if(rule.operator === '=' && answer.value !== rule.value) {
-            returnValue = false;
-          }else if(rule.operator === '!=' && answer.value === rule.value){
-            returnValue = false;
-          }else if(rule.operator === '>' && answer.value <= rule.value){
-            returnValue = false;
-          }else if(rule.operator === '<' && answer.value >= rule.value){
-            returnValue = false;
+      var ruleOperand = block.content && block.content.ruleoperand ? block.content.ruleoperand : 'AND';
+      var validRuleCount = 0;
+
+      for (var i = 0; i < block.rules.length; i++) {
+        var rule = block.rules[i];
+        var answer = _.find(answers, { blockId: rule.parentBlockId });
+        if (angular.isDefined(answer) && answer.value !== '') {
+          if (rule.operator === '=' && answer.value === rule.value) {
+            validRuleCount++;
+          } else if (rule.operator === '!=' && answer.value !== rule.value) {
+            validRuleCount++;
+          } else if (rule.operator === '>' && answer.value > rule.value) {
+            validRuleCount++;
+          } else if (rule.operator === '<' && answer.value < rule.value) {
+            validRuleCount++;
           }
         }
-      });
+        if (ruleOperand === 'OR' && validRuleCount > 0) {
+          break;
```

Comment:
You could also `break` if `ruleOperand === 'AND' && validRuleCount <= i` cuz we know that if any rule isn't met, the whole `AND` can't be true.